packages/terra-section-header/docs/README.md

```diff
@@ -0,0 +1,37 @@
+# Terra Section Header
+
+Section Header presentational component that provides an onClick callback and animatable status icon.
```

Comment:
nit: Drop the `and animateable status icon.` part since we scoped that out.

Reply:
Done 
aa42014004dd17cacbdf78f5176b2a74dcfbff6a